Lib/fontmake/compatibility.py

```diff
@@ -71,7 +70,7 @@ def ensure_all_same(self, func, objs, what):
             return True
         logger.error(f"Fonts had differing {what} in {context}:")
         for value, fonts in values.items():
-            if self.verbose or len(fonts) <= 6:
+            if logger.isEnabledFor(logging.DEBUG) or len(fonts) <= 6:
```

Comment:
move this `logger.isEnabledFor(logging.DEBUG)` check out of the for loop please, it's constant throughout the loop. Save it to a local variable and use in the loop